testsuite/tests/typing-unique/unique.ml

```diff
@@ -544,7 +541,7 @@ let curry : unique_ box -> (unique_ box -> unique_ box) = fun b1 b2 -> b1
 Line 1, characters 58-73:
 1 | let curry : unique_ box -> (unique_ box -> unique_ box) = fun b1 b2 -> b1
                                                               ^^^^^^^^^^^^^^^
-Error: This function or one of its parameters will be at once but expected to be many
+Error: This function or one of its parameters will be once but will be used many
```

Comment:
I have incorporated the first option because it's probably better to treat the `linearity` case of `Uncurried_function_escape` in the same way as the other axes, and for those axes we don't print the whole type signature.